tests/components/wled/test_sensor.py

```diff
@@ -94,6 +98,7 @@ async def test_sensors(
     assert state.attributes.get(ATTR_LED_COUNT) == 30
     assert state.attributes.get(ATTR_MAX_POWER) == 850
     assert state.attributes.get(ATTR_UNIT_OF_MEASUREMENT) == CURRENT_MA
+    assert state.attributes.get(ATTR_DEVICE_CLASS) == DEVICE_CLASS_CURRENT
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5985897)